context/app/static/js/components/Header/DocumentationLinks/DocumentationLinks.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DropdownLink from '../DropdownLink';
+
+function DocumentationLinks(props) {
+  const { isIndented } = props;
+  return (
+    <>
+      <DropdownLink href="/docs" isIndented={isIndented}>
+        Overview
+      </DropdownLink>
+      {['FAQ', 'About'].map((name) => (
+        <DropdownLink key={name} href={`/docs/${name.toLowerCase()}`} isIndented={isIndented}>
+          {name}
+        </DropdownLink>
+      ))}
```

Comment:
```suggestion
      <DropdownLink href="/docs/faq" isIndented={isIndented}>
        FAQ
      </DropdownLink>
      <DropdownLink href="/docs/about" isIndented={isIndented}>
        About
      </DropdownLink>
```
Keeping the status quo is fine too, but if there are changes going forward, we shouldn't get stuck trying to cram more ad-hoc logic into this.